learn/guides/graphql-client-tool.md

```diff
@@ -59,7 +93,46 @@ This will generate the files below.
 2. `utils.bal` - util file for the relevant `utils` methods related to the client stub
 3. `types.bal`- schema file for the configured GraphQL schema
 
->**Note:** If the GraphQL API contains an authentication mechanism, add the extensions section in the GraphQL config file with the relevant tokens and headers. In this scenario, it is mandatory to configure the schema section with the web URL of the GraphQL schema as shown below.
+For the example above, it would generate the following client file.
+
+```ballerina
+import ballerina/http;
+import ballerina/graphql;
+
+public isolated client class QuerycountryClient {
+    final graphql:Client graphqlClient;
+    public isolated function init(string serviceUrl, http:ClientConfiguration clientConfig = {}) returns graphql:ClientError? {
+        graphql:Client clientEp = check new (serviceUrl, clientConfig);
+        self.graphqlClient = clientEp;
+        return;
+    }
+    remote isolated function countryByCode(string code) returns CountryByCodeResponse|graphql:ClientError {
+        string query = string `query countryByCode($code:ID!) {country(code:$code) {name}}`;
+        map<anydata> variables = {"code": code};
+        json graphqlResponse = check self.graphqlClient->executeWithType(query, variables);
+        return <CountryByCodeResponse> check performDataBinding(graphqlResponse, CountryByCodeResponse);
+    }
+}
+```
+
+We recommend copying the generated files into the root of a Ballerina package library, so that you can publish the client package to Ballerina central and reuse the client package in another Ballerina program. For more information, please refer to the [publishing package to Ballerina central](/learn/publish-packages-to-ballerina-central/).
```

Comment:
```suggestion
It is recommended to copy the generated files into the root of a Ballerina package library so that you can publish the client package to Ballerina Central and reuse the client package in another Ballerina program. For more information, see [Publish packages to Ballerina Central](/learn/publish-packages-to-ballerina-central/).
```